### PR TITLE
Add basic UI with the CLI

### PR DESCRIPTION
**Pull request by Alabi12**
# Summary

- [ ] Create a app.rb file that will serve as your console app entry-point. It should have methods that do the following:

- List all books.
- List all people.
- Create a person (teacher or student, not a plain Person).
- Create a book.
- Create a rental.
- List all rentals for a given person id.

- [ ] In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:

- Present the user with a list of options to perform.
- Lets users choose an option.
- If needed, ask for parameters for the option.
- Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    else
+      @people.each do |person|
+        puts "[#{person.class.name}] Name: #{person.name} ID: #{person.id} Age: #{person.age}"
+      end
+    end
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input a number]:'
+    person_type = gets.chomp.to_i
```
Comment (Munsa1) [inline app.rb:39]: - [ ] When the user tries to create a student the following error occurs, kindly fix this issue.
image

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    books.each_with_index do |book, index|
+      puts "#{index}) Title: #{book.title} Author: #{book.author}"
+    end
+    book_index = gets.chomp.to_i
+    return invalid_input if book_index > books.length
+
+    puts 'Select a person from the following list by number (not id): '
+    people.each_with_index do |person, index|
+      puts "#{index}) [#{person.class.name}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
```
Comment (Munsa1) [inline app.rb:103]: - [ ] When the user tries to create a rental, the following error occurs. Kindly fix this.
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Alabi12, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly capitalize the first words of the commit messages going forward.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (DeliceLydia) [reply app.rb:39]: Addressed

File: app.rb (hunk 2 above)
Comment (DeliceLydia) [reply app.rb:103]: Addressed

**Review by DeliceLydia — APPROVED** [review]
## Hi @Alabi12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Codes are well-organized and clean
- Followed gitworkflow


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**